templates/footer.hbs

```diff
@@ -14,27 +14,48 @@
       </div>
   {{/if}}
 
-  <div class="container">
-      <div class="footer-top">
-        <div class="mit-info">
-          <img src="{{settings.mit_footer_logo}}" alt="MIT" loading="lazy"/>
+<footer class="new-footer">
+  <div class="new-footer-container">
+      <div class="copyright-container">
+          <div class="address-col">
+            <a href="https://openlearning.mit.edu">
+              <img src="{{settings.mit_footer_logo}}" alt="MIT" loading="lazy"/>
+            </a>
             <address>{{settings.mit_address}}</address>
```

Comment:
I am not 100% sure but @pdpinch asked me to update the address when I was working on the Bootcamps logo update https://github.com/mitodl/bootcamp-ecommerce/pull/1484#pullrequestreview-1866777694. Also, this might need to be updated in the MITx Online app footer.